src/praisonai/praisonai/llm/credentials.py

```
@@ -135,44 +186,48 @@ def is_configured(model: Optional[str] = None) -> bool:
+    if required_vars:
+        if any(os.environ.get(v) for v in required_vars):
+            return True
+        # Check stored credentials for the matching provider only.
+        try:
+            store = CredentialStore()
+            providers = [p.lower() for p in store.list_providers()]
+            wanted = _stored_providers_for_vars(required_vars)
+            if any(p in providers for p in wanted):
+                return True
+        except Exception:
+            pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)